php/libraries/Candidate.class.inc

```diff
@@ -657,17 +657,52 @@ class Candidate
     }
 
     /**
-     * Returns the candidate's calculated age today
+     * Returns the candidate's calculated age today in years, months, days
      *
-     * @return int The candidate's age today
+     * @return array The candidate's age today in an array with values for keys 'year',
+     *               'mon', and 'day'
      */
-    public function getAge(): int
+    public function getAge(): array
     {
-        $ageArray = \Utility::calculateAge(
+        return \Utility::calculateAge(
             $this->candidateInfo["DoB"],
             (new DateTime())->format('Y-m-d')
         );
-        return $ageArray['year'];
+    }
+
+    /**
+     * Returns the candidate's calculated age today in int years
+     *
+     * @return int The candidate's age today as years
+     */
+    public function getAgeYears(): int
```

Comment:
```suggestion
    public function getAgeInYears(): int
```

Just for consistency

Reply:
this is done deliberately to mean that its the year or your age vs. your age in year i.e. 2.25 years as opposed to just 2 years

Reply:
Whether the word "In" is there or not doesn't make it clear that you're returning an integer year vs. a decimal year. 

The other functions have "In" in the name and also return integers.

Reply:
I second @johnsaigle argument about the fact that the others 2 functions have `in` in the name. Should be all with or without for consistency.

Reply:
I agree with the others, this is confusingly inconsistent with the other functions